Spanner/src/Database.php

```diff
@@ -405,7 +406,12 @@ public function create(array $options = [])
         $statements = $this->pluck('statements', $options, false) ?: [];
 
         $databaseId = DatabaseAdminClient::parseName($this->name())['database'];
-        $statement = sprintf('CREATE DATABASE `%s`', $databaseId);
+        if(isset($options['databaseDialect']) && $options['databaseDialect'] === DatabaseDialect::POSTGRESQL){
```

Comment:
I could abstract this in a prvt function 🤔

Reply:
I had the same thought... I think it would be better!